pkg/download/download.go

```diff
@@ -86,39 +90,55 @@ func (p *protocol) Latest(ctx context.Context, mod string) (*storage.RevInfo, er
 
 func (p *protocol) GoMod(ctx context.Context, mod, ver string) ([]byte, error) {
```

Comment:
I'd like to recommend a different name, but can't think of a better one. At first I thought `Mod` but that may be confused for the _entire_ module. Then maybe `ModInfo` but that would seem to represent the module's `RevInfo`. So good as is I guess :grin: